security/production-secure-deploy/README.rst

```diff
@@ -196,13 +196,24 @@ Deploy Confluent Platform
    
      kubectl get pods
 
-Note: The default required RoleBindings for each Confluent component are created
+If any component does not deploy, it could be due to missing configuration information in secrets.
+The Kubernetes events will tell you if there are any issues with secrets. For example:
+
+   ::
+
+     kubectl get events
+     Warning  KeyInSecretRefIssue  kafka/kafka  required key [ldap.txt] missing in secretRef [credential] for auth type [ldap_simple]
+
+The default required RoleBindings for each Confluent component are created
 automatically, and maintained as `confluentrolebinding` custom resources.
 
    ::
 
      kubectl get confluentrolebinding
-   
+
+If you'd like to see how the RoleBindings custom resources are structured, so that
+you can create your own RoleBindings, take a look at the custom resources in this 
+directory: $TUTORIAL_HOME/internal-rolebindings
```

Comment:
should we rename the internal-rolebindings to some name like rolebinding-examples?